salt/knox/files/yarnui_rewrite.xml

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Please add a comment linking to where this came from and the tracking information. We should have a JIRA linking to the upstream JIRA that closes this and references the upstream release that will contain the fix. We will then bring in that version and remove this.